PythonDZ1/DZ1.1.py

```diff
@@ -0,0 +1,11 @@
+#адача 2: Найдите сумму цифр трехзначного числа.
+#*Пример:*
+#123 -> 6 (1 + 2 + 3)
+#100 -> 1 (1 + 0 + 0) 
+
+a=int(input('Введите трехзначное число:'))
+
+summa=(a//100)+((a//10)-a//100*10)+a%10
```

Comment:
Тут нигде скобки не нужны
```suggestion
a = int(input('Введите трехзначное число:'))

summa = a // 100 + a // 10 - a // 100 * 10 + a % 10
```